### ISSUE TITLE
replenishDefaultLicenseTexts flag makes the output file unavailable when using react-jss

### ISSUE THREAD

**Issue by zoltan-mihalyi** [codepunkt/webpack-license-plugin#428]
I noticed a weird bug: when I set replenishDefaultLicenseTexts: true and use react-jss package, the output file becomes unavailable.

Try the attached example, run "npm run dev" and open "localhost:9000/out.json" which gives 404.
When I remove react-jss or replenishDefaultLicenseTexts, it is working again.

[package.zip](https://github.com/codepunkt/webpack-license-plugin/files/5377885/package.zip)

**Comment by zoltan-mihalyi** [comment]
It seems like replacing `compilation.hooks.optimizeChunkAssets.tap` call with `compilation.hooks.optimizeChunkAssets.tapAsync` solves the problem.

**Comment by codepunkt** [comment]
Thanks Zoltan, I'll see when i can get to that. What version of webpack are you using?

**Comment by zoltan-mihalyi** [comment]
I use these in the attached project:

```json
{
    ...
    "webpack": "^4.44.2",
    "webpack-cli": "^3.3.12",
    "webpack-dev-server": "^3.11.0",
    ...
}
```

**Comment by zoltan-mihalyi** [comment]
Should I create a PR? I'm afraid I can't write tests, though.

**Comment by zoltan-mihalyi** [comment]
Someone made a PR before me, can you merge this, @codepunkt ?
https://github.com/codepunkt/webpack-license-plugin/pull/423

**Comment by zoltan-mihalyi** [comment]
@codepunkt can you publish a hotfix version?

**Comment by codepunkt** [comment]
@zoltan-mihalyi published as `4.1.2`, thanks!
